### DISCUSSION TITLE
How to Build an Open Source community

### DISCUSSION THREAD

**Discussion by Ntshangase** [community/community#49673] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

Hello,

How best can I promote my Open Source project?, I want people to know about this amazing project so that they can use it and  contribute if they want to add something.

The project : https://github.com/Ntshangase/GitHub-Profile-Builder

**Comment by bkpecho** [comment]
Hey @Ntshangase 

I can think of some proven tips to promote your open-source project and get the word out to potential users and contributors:

- Write a clear and concise README for your project
- Promote your project on social media, such as Twitter or Reddit
- Submit your project to relevant directories, such as GitHub Showcase or Awesome lists
- Reach out to potential contributors or users directly, such as through email or messaging platforms
- Collaborate with other open-source projects or organizations to cross-promote each other's work

Thank you for supporting open-source causes and I've already starred your project!

Hope my answer helps. Have a nice day!
